fraud/helpers.go

```diff
@@ -2,19 +2,30 @@ package fraud
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/protocol"
 	pubsub "github.com/libp2p/go-libp2p-pubsub"
 )
 
-func join(p *pubsub.PubSub, proofType ProofType,
+func pubSubTopicID(fraudType, protocolSuffix string) string {
+	return "/fraud-sub/" + fraudType + "/v0.0.1/" + protocolSuffix
```

Comment:
[non-blocking] we should be consistent everywhere with this pattern (im counting this as part of the protocolID func pattern) and use fmt.Sprintf (but it would also have to be fixed in other places, so I will create an issue)